jekyll/_cci2/gh-bb-integration.md

```diff
@@ -65,6 +65,17 @@ Integrated status also appears on the pull request screen, to show that all test
     3. Regenerate the keys in CircleCI project.
 - Ensure no developer has access to a build in a repo with a User Key that requires more access than they have.
 
+## Renaming Orgs and Repositories
+
+If you find you need to rename an org or repo that you have previously hooked up to CircleCI, best practice is to follow these steps:
+
+1. Rename org/repo in VCS
+2. Head to the CircleCI application, using the new org name
```

Comment:
Can we give a format for the URL here? I guess `app.circleci.com/pipelines/<VCS>/<new-org-name>/<project-name>` from looking at my account, but maybe there's something simpler...?

Reply:
I believe that's the right URL to use.